custom/icds_reports/migrations/0156_auto_20191217_0932.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.26 on 2019-12-17 09:32
+from __future__ import unicode_literals
+
+import custom.icds_reports.models.aggregate
+from django.db import migrations, models
+
+from custom.icds_reports.utils.migrations import (
+    get_composite_primary_key_migrations,
+)
+from corehq.sql_db.operations import RawSQLMigration
+from corehq.sql_db.connections import is_citus_db
+from custom.icds_reports.const import AGG_ADOLESCENT_GIRLS_REGISTRATION_TABLE
+from custom.icds_reports.utils.migrations import (
+    create_citus_distributed_table
+)
+from custom.icds_reports.utils.migrations import get_view_migrations
+
+migrator = RawSQLMigration(('custom', 'icds_reports', 'migrations', 'sql_templates', 'database_views'))
+
+
+def _distribute_citus_tables(apps, schema_editor):
+    with schema_editor.connection.cursor() as cursor:
+        if not is_citus_db(cursor):
+            return
+
+        for table, col in [(AGG_ADOLESCENT_GIRLS_REGISTRATION_TABLE, 'supervisor_id')]:
+            create_citus_distributed_table(cursor, table, col)
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('icds_reports', '0155_added_new_fields_css_record_monthly_and_delivery_forms'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='AggregateAdolescentGirlsRegistrationForms',
+            fields=[
+                ('person_case_id', models.TextField(primary_key=True, serialize=False)),
+                ('state_id', models.TextField(null=True)),
+                ('supervisor_id', models.TextField(null=True)),
+                ('awc_id', models.TextField(null=True)),
+                ('out_of_school', models.NullBooleanField()),
+                ('re_out_of_school', models.NullBooleanField()),
+                ('admitted_in_school', models.NullBooleanField()),
+                ('month', models.DateField(null=True)),
+            ],
+            options={
+                'db_table': 'icds_dashboard_adolescent_girls_registration',
+            },
+            bases=(models.Model, custom.icds_reports.models.aggregate.AggregateMixin),
+        ),
+        migrations.AlterUniqueTogether(
+            name='aggregateadolescentgirlsregistrationforms',
+            unique_together=set([('supervisor_id', 'person_case_id', 'month')]),
```

Comment:
If this creates an index in that order, I think you want month first, since that will be the biggest limiter

Reply:
hey sorry did not follow.  why is that order matter? This just ensures that these together are unique no? Also creates a index like this `"icds_dashboard_adolescen_supervisor_id_person_cas_de4d929b_uniq" UNIQUE CONSTRAINT, btree (supervisor_id, person_case_id, month).` How is the order matter of this matter?